### PR TITLE
MR-1816: Add and update app permissions for mc review business owner

### PR DESCRIPTION
**Pull request by JasonLin0991**
## Summary
[MCR-1816](https://qmacbis.atlassian.net/browse/MCR-1816)

- Add `BUSINESSOWNER_USER` type to the app that maps to the idm role `macmcrrs-bo`
- Set the local login user cards to be width of 200px

### CODE REVIEW

File: services/app-api/prisma/migrations/20230929194337_add_business_owner_user_role/migration.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+BEGIN;
+-- AlterEnum
+ALTER TYPE "Role" ADD VALUE 'BUSINESSOWNER_USER';
+
+-- AlterTable
+ALTER TABLE "UserAudit" ALTER COLUMN "updatedAt" DROP DEFAULT;
```
Comment (JasonLin0991) [inline services/app-api/prisma/migrations/20230929194337_add_business_owner_user_role/migration.sql:6]: @macrael I coped this line from your merged [PR](https://github.com/Enterprise-CMCS/managed-care-review/pull/1791) that added the helpdesk user. I'm guessing this is still needed here?

File: services/app-graphql/src/schema.graphql [hunk 2]
```diff
@@ -462,6 +462,16 @@ type HelpdeskUser {
     familyName: String!
 }
 
+"BusinessOwnerUser is a user that ..."
```
Comment (JasonLin0991) [inline services/app-graphql/src/schema.graphql:465]: Halp! What exactly do we want to be the description here?

**Review by JasonLin0991 — COMMENTED** [review]
Couple of things I had questions on.

File: services/app-graphql/src/schema.graphql (hunk 2 above)
Comment (haworku) [inline services/app-graphql/src/schema.graphql:465]:
```suggestion
"BusinessOwnerUser is a CMS user that manages CMS and program level settings in the app."
```

File: services/app-web/src/localAuth/LocalLogin.tsx [hunk 3]
```diff
@@ -90,6 +99,7 @@ const userAvatars: { [key: string]: string } = {
     'roku@example.com': rokuAvatar,
     'izumi@example.com': izumiAvatar,
     'appa@example.com': appaAvatar,
+    'shi-tong@example.com': shiAvatar,
```
Comment (haworku) [inline services/app-web/src/localAuth/LocalLogin.tsx:102]: approved 👍

**Review by haworku — APPROVED** [review]
Noticing that there seems to be a group of user roles that have similar permissions -   e.g, they can access settings panel (logic like `isAdminUser || isHelpdeskUser || isBusinessOwnerUser`). Can we encode that in domain model helper `canViewSettings` or something like that.

File: services/app-api/prisma/migrations/20230929194337_add_business_owner_user_role/migration.sql (hunk 1 above)
Comment (macrael) [reply services/app-api/prisma/migrations/20230929194337_add_business_owner_user_role/migration.sql:6]: That default should already be dropped so I don't think it makes sense to drop it again.

File: services/app-graphql/src/schema.graphql (hunk 2 above)
Comment (macrael) [reply services/app-graphql/src/schema.graphql:465]: The Business Owner is the root of our role hierarchy with the power to approve new admin accounts

**Comment by JasonLin0991** [comment]
> Noticing that there seems to be a group of user roles that have similar permissions - e.g, they can access settings panel (logic like `isAdminUser || isHelpdeskUser || isBusinessOwnerUser`). Can we encode that in domain model helper `canViewSettings` or something like that.

How about this?

```ts
function hasAdminPermissions(user: UserType): boolean {
    const authorizedAdmins = [
        'ADMIN_USER',
        'HELPDESK_USER',
        'BUSINESSOWNER_USER',
    ]

    return authorizedAdmins.includes(user.role)
}
```

**Pull request merged**